source/routes/update/month.js

```diff
@@ -0,0 +1,51 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7423478)
```suggestion
PouchDB.plugin(require("pouchdb-find"));
```